app/src/main/java/com/sun_asterisk/foodies/screen/main/MainActivity.kt

```diff
@@ -2,12 +2,40 @@ package com.sun_asterisk.foodies.screen.main
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.viewpager.widget.ViewPager
 import com.sun_asterisk.foodies.R
+import com.sun_asterisk.foodies.screen.fragment.FavoriteFragment
+import com.sun_asterisk.foodies.screen.fragment.HomeFragment
+import com.sun_asterisk.foodies.screen.main.layout_adapter.ViewPagerAdapter
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        setupTabLayout()
+    }
+
+    private fun setupViewPager(viewPager: ViewPager) {
+        val adapter = ViewPagerAdapter(supportFragmentManager)
+        adapter.addFrag(HomeFragment(), resources.getString(R.string.home))
+        adapter.addFrag(FavoriteFragment(), resources.getString(R.string.favorite))
+        viewPager.adapter = adapter
+    }
+
+    private fun setupTabIcons() {
+        val icons = resources.obtainTypedArray(R.array.layout_icons)
+        for (i in 0..1) {
+            tabs.getTabAt(i).let { it?.setIcon(icons.getDrawable(i)) }
+        }
```

Comment:
use when or switch case
Vì chỉ có 2 item
https://stackoverflow.com/questions/21354501/typed-array-should-be-recycled-after-use-with-recycle